microsoft/PowerToys
# [Hosts] Revert content dialog work-around for titlebar overlap

## Summary of the Pull Request

Revert the work-around applied to content dialogs for titlebar overlap.


## PR Checklist

- [x] **Closes:** #26647


## Detailed Description of the Pull Request / Additional comments

<img width="351" alt="0 64 0" src="https://github.com/microsoft/PowerToys/assets/25966642/ebf34de1-fe56-4e14-851c-83d3a14e653c">

_0.64.0 bugged_

<img width="351" alt="0 70 1" src="https://github.com/microsoft/PowerToys/assets/25966642/7461d482-5667-4043-a976-0475f621da80">

_0.70.1_

<img width="351" alt="PR" src="https://github.com/microsoft/PowerToys/assets/25966642/83850829-4c43-4fd1-8f84-5bae8756f58f">

_PR_


## Validation Steps Performed

- Build
- Start Hosts
- Resize the window at minumum size
- Verify that entry dialog and additional content dialog are not overlapping with titlebar
